Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.14 AS builder
+FROM registry.ci.openshift.org/ocp/builder:golang-1.15 AS builder
```

Comment:
Might make sense to do 

registry.ci.openshift.org/origin/builder:golang-1.15

That requires less authentication than registry.ci.openshift.org/ocp/builder:golang-1.15 and is consisent with 
registry.ci.openshift.org/openshift/origin-v4.0:base

Where as the Dockerfile.rhel file uses the `ocp` namespace, and that requires authentication credentials.

Note, the Dockerfile.rhel is already doing 1.16 ... https://github.com/openshift/openshift-controller-manager/blob/master/Dockerfile.rhel#L1 but when I tried to pull

registry.ci.openshift.org/origin/builder:golang-1.16

it does not exist yet.

Reply:
It seems `Dockerfile` can be deleted. Both openshift/release and ocp-build-data use `Dockerfile.rhel`.